components/GardenPlant.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { View, Text, TouchableHighlight} from 'react-native';
+import { View, Text, TouchableOpacity} from 'react-native';
```

Comment:
I think this was a good change over from TouchableHighlight

Reply:
Ya, I think it looks better. We should change the one on the Search page as well.